changelog.json

```
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.7.0-beta7": [
+      "[Improved] Error message when unable to publish private repository to an organization - #7472",
+      "[Improved] \"View Stash\" button accessibility improvements - #7274",
```

Isn't this the button in the changes panel?
```suggestion
      "[Improved] \"Stashed changes\" button accessibility improvements - #7274",
```

---

ah yes good point